nodes/mainnet.md

```diff
@@ -153,6 +154,7 @@ to participate in Mainnet Beta:
 - `api-celestia.staker.space`
 - `celestia-api.noders.services`
 - `celestia.moonli.me/api`
+- `celestia-mainnet-api.itrocket.net`
```

Comment:
what ports are open?

Reply:
443

Reply:
could you add this in bullets underneath please?